src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -313,19 +228,26 @@ public synchronized ListeFilme neueFilme(ListeFilme orgListe) {
         return ret;
     }
 
-    public String getFileSizeUrl(String url, String sender) {
+    /**
+     * @param url the URL as String.
+     * @return the determined size or -1.
+     * @deprecated Move this someday to DatenFilm.
+     */
+    @Deprecated
+    public String getFileSizeUrl(String url) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1133)